packages/cli/src/commands/dev.ts

```diff
@@ -31,10 +32,21 @@ const defaultIgnored = [
 
 const devAbortController = new AbortController()
 
-async function storeDev(rootDir: string, tmpDir: string) {
+async function storeDev(rootDir: string, tmpDir: string, coreDir: string) {
   const envVars = dotenv.parse(readFileSync(path.join(rootDir, 'vtex.env')))
 
-  const devProcess = spawn('yarn dev', {
+  runCommandSync({
+    cmd: 'yarn predev',
+    errorMessage:
+      'GraphQL was not optimized and TS files were not updated. Changes in the GraphQL layer did not take effect',
+    throws: 'error',
+    debug: false,
```

Comment:
IMO we should enable debug by default or have a way to switch this easily to true.

Reply:
Will incorporate your feedback!

Reply:
To keep the same behavior as before, I've fixed the debug to `true`